python/ray/train/v2/api/exceptions.py

```diff
@@ -6,7 +6,7 @@
 
 @PublicAPI(stability="alpha")
 class TrainingFailedError(RayTrainError):
-    """Exception raised by `<Framework>Trainer.fit()` when training fails."""
+    """Exception raised after the controller finishes and training starts."""
```

Comment:
The controller will be active throughout the training, so would be good to clarify what this means - is it more accurate to say that it's an exception that's raised from one or more of the distributed training workers?